lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.t

```diff
@@ -36,7 +36,7 @@ symlink($reference_fasta_index,$reference_build->data_directory .'/all_sequences
 
 my $result_users = Genome::Test::Factory::SoftwareResult::User->setup_user_hash(reference_sequence_build_id => $reference_build->id,);
 
-my $params = "-g,-d,-o:Hello";
+my $params = '-P,-g,-d';
```

Comment:
I agree testing the syntax would be useful, but the param did not work in the first place.  The -o option is to define the output_prefix which is redefined here:
https://github.com/jasonwalker80/genome/blob/28d3c86f0c66f242abf3b343ebacb46f427cc46b/lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.pm#L46